modules/get-started/pages/whats-new.adoc

```diff
@@ -2,76 +2,25 @@
 :description: Summary of new features and updates in the release.
 :page-aliases: get-started:whats-new-233.adoc, get-started:whats-new-241.adoc
 
-This topic includes new content added in version 24.1. For a complete list of all product updates, see the https://github.com/redpanda-data/redpanda/releases/[Redpanda release notes^]. See also: 
+This topic includes new content added in version 24.2 beta. For a complete list of all product updates, see the https://github.com/redpanda-data/redpanda/releases/[Redpanda release notes^]. See also: 
 
 * xref:deploy:deployment-option/cloud/whats-new-cloud.adoc[] 
 * xref:deploy:deployment-option/cloud/cloud-overview.adoc#redpanda-cloud-vs-self-hosted-feature-compatibility[Redpanda Cloud vs self-hosted feature compatibility]
 
-== Redpanda Connect 
+== FIPS compliance
 
-In release 24.1.5, Redpanda announced xref:redpanda-connect:ROOT:about.adoc[Redpanda Connect]. This includes over 220 open source connectors, with a large collection of sources, sinks, and processors. The interface provides a lightweight and easy way to set up, debug, and manage connectors. 
+Redpanda now provides FIPS-compliant cipher enforcement for brokers using OpenSSL. Both Redpanda and `rpk` leverage validated OpenSSL libraries for all security-related cryptography operations. Install the `redpanda-fips` package to xref:manage:security/fips-compliance.adoc[run Redpanda in FIPS compliance mode].
```

Comment:
@kbatuigas 

let's go with:

"Redpanda now provides a FIPS 140 compliant binary, to support running Redpanda in environments requiring FIPS compliant software, and to meet certain requirements for customers undergoing certification through government compliance programs such as FedRAMP.  When using the FIPS binary, both Redpanda and rpk leverage FIPS-compliant OpenSSL libraries & ciphers for all encryption activity inside Redpanda including encryption of data in transit.  Install the redpanda-fips package to run Redpanda in FIPS compliance mode"

Reply:
@kbatuigas  on second thought, 'users' should be 'organizations'.   Besides that looks good